module/packer/signer_indices.go

```diff
@@ -0,0 +1,65 @@
+package packer
+
+import (
+	"fmt"
+)
+
+// EncodeSignerIndices encodes indices into compacted bit vector. Each bit represents whether the identity at
+// that index is a signer.
+// Note, the indices in the first argument must be in the strict increasing order,
```

Comment:
if we re-use the tools from `ledger/utils`, I think this condition won't be required anymore.